src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const keys of action.keysToRemove) {
```

Comment:
there's just a single key
```suggestion
        for (const key of action.keysToRemove) {
```